### DISCUSSION TITLE
[Q&A] `AIRFLOW__METRICS__METRICS_ALLOW_LIST` values?

### DISCUSSION THREAD

**Discussion by seunggabi** [apache/airflow#33243] [Q&A]
I found `AIRFLOW__METRICS__METRICS_ALLOW_LIST` items.
But I can find `scheduler,executor,dagrun`.
Just all three things?

https://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html

<img width="500" alt="스크린샷 2023-08-09 오전 11 08 55" src="https://github.com/apache/airflow/assets/8468371/fb8bfc20-43bc-4195-bdd3-6ac4d863d021">

**Comment by seunggabi** [comment]
I know that `AIRFLOW__METRICS__METRICS_ALLOW_LIST == ''` is allowed all things.

**Comment by Stephan-adj** [comment]
Hello there, I know those values works : 
`value: "scheduler,executor,dagrun,pool,triggerer,celery"`

but if someone could provide us more insights on this var I would also definitely be interested !
@raphaelauv any ideas ? :)

**Reply by Stephan-adj** [reply]
I also found this very useful : https://github.com/apache/airflow/blob/205ad57d040201c5b0690d100719207695028fc2/docs/apache-airflow/administration-and-deployment/logging-monitoring/metrics.rst#allowblock-lists
